src/routes/console/project-[project]/storage/+page.svelte

```diff
@@ -1,31 +1,34 @@
+<script lang="ts" context="module">
+    export let showCreateBucket = writable(false);
```

Comment:
shouldn't `writeable` vars be part of store.ts?

Reply:
Not in this case, as this only affects something directly present in this page. The stores in store.ts usually contain information pertinent to all its children